### ISSUE TITLE
Training collapse with NaN loss using pretrained autoencoder

### ISSUE THREAD

**Issue by tliby** [bytetriper/RAE#16]
Awesome work 👏 — thanks for releasing this great project and pretrained models!

I followed the instructions to use the provided pretrained autoencoder as part of training lightningDiT. However, during training the loss suddenly becomes NaN, and the samples generated afterward turn completely black.

Try1:
<img width="671" height="182" alt="Image" src="https://github.com/user-attachments/assets/4d71c356-a2db-4f93-bd48-88ece15ad826" />

Try2:

<img width="721" height="228" alt="Image" src="https://github.com/user-attachments/assets/1bb42597-ec23-4f4c-b8c2-14269977eae5" />

Script:
torchrun --standalone --nnodes=1 --nproc_per_node=8 \
  src/train.py \
  --config configs/stage2/training/ImageNet256/DiTDH-XL_DINOv2-B.yaml \
  --precision bf16

Hyperparemeter:
  global_seed: 0
  epochs: 80
  global_batch_size: 512
  grad_accum_steps: 2
  decay_start_epoch: 40
  decay_end_epoch: 80

Could you please help me understand why the training collapses with NaN loss? Thank you in advance for your time and assistance!
